bubbleseekbar/src/main/java/com/xw/repo/BubbleSeekBar.java

```diff
@@ -1158,6 +1159,19 @@ private float processProgress() {
         return progress;
     }
 
+    public void setProgressText(@NonNull String text) throws ProgressTextLengthException {
```

Comment:
The progress text will be updated only the text is not null, is not empty and length is less or equal than 5